package_gd32_arcayi_index.json

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
Can we exclude this file from the merge? Otherwise it looks very good now.

Reply:
fixed